src/app/screens/verify/verify.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, OnInit } from "@angular/core";
+import { Router, ActivatedRoute } from "@angular/router";
+import { AppService } from "src/app/services/app.service";
+import { CookieService } from "ngx-cookie-service";
+@Component({
+  selector: "app-verify",
+  templateUrl: "./verify.component.html",
+  styleUrls: ["./verify.component.css"]
+})
+export class VerifyComponent implements OnInit {
+  isLoading: boolean = true;
+  userName: string;
+  cookieValue: string;
+  cookieAvailable: boolean;
+  constructor(
+    private router: Router,
+    private appservice: AppService,
+    private activatedRoute: ActivatedRoute,
+    private cookie: CookieService
+  ) {}
+  role: any;
+  ngOnInit() {
+    this.appservice.currentMessage.subscribe(name => {
+      this.userName = name
+    });
+
+    this.cookieAvailable = this.cookie.check("ttemail");
+    if (this.cookieAvailable == true) {
+      this.appservice
+        .getEmployeeRole(this.cookie.get("ttemail"))
+        .subscribe(response => {
+          if(response.code == "00"){
+            var existtingUserData = response.data;
+            if(Object.keys(response.data).length === 0) {
+              var employee_email = this.cookie.get("ttemail");
+              var employee_firstname = this.cookie.get("userFirstName");
+              var employee_lastname = this.cookie.get("userlastName");
+              let requestData = {
+                "employee_address": "",
+                "employee_dev_level": "",
+                "employee_email": employee_email,
+                "employee_firstname": employee_firstname,
+                "employee_gender": "",
+                "employee_lastname": employee_lastname,
+                "employee_phonenumber": "",
+                "employee_role": "developer",
+                "employee_status": ""
+              }
+
+              this.appservice
```

Comment:
let's build a front-end employee service and a project service and separate out this logic